spec/features/admin_items_view_spec.rb

```diff
@@ -1,38 +1,27 @@
 require 'rails_helper'
 
 describe "the item view", type: :feature do
-  let(:item) do
-    Item.create(title: "Possum Stew",
-                description: "Savory stew made with fresh possum",
-                image: "www.possum.com",
-                item_status_id: 1,
-                price: 2.35)
-  end
-
-  let(:item2) do
-    Item.create(title: "Possum2 Stew",
-                description: "Savory stew made with fresh possum",
-                image: "www.possum.com",
-                item_status_id: 1,
-                price: 2.35)
+  before :each do
+    @item = FactoryGirl.create(:item)
+    admin = FactoryGirl.create(:admin)
+    visit login_path
+    fill_in("session_email", with: admin.email)
+    fill_in("session_password", with: admin.password)
+    click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.